q.js

```diff
@@ -1645,10 +1645,10 @@ function any(promises) {
         function onRejected(err) {
             pendingCount--;
             if (pendingCount === 0) {
-                var rejection = err || new Error("Rejection was null/undefined");
+                var rejection = err || new Error("Rejection value was: " + err);
```

Comment:
Thinking more, do you think it might not suffice to just toString the error instead of prefixing with any front-matter? `("" + err)`. Consider ultimately `Q.any can't get fulfillment value from any promise. All promises were rejected. Last error: undefined`.